password-generator@spencerlommel.com/README.md

```diff
@@ -0,0 +1,14 @@
+Installation
+============
+
+Using a terminal, you can do the following sequence of commands:
+
+```
+git clone https://github.com/SpencerLommel/cinnamon_password_generator.git
+mkdir -p ~/.local/share/cinnamon/applets/password-generator@spencerlommel.com/ && mv cinnamon_password_generator/* ~/.local/share/cinnamon/applets/password-generator@spencerlommel.com/
+rm -rf cinnamon_password_generator
+```
+
+Then press `Alt`+`F2`, type `r` in the dialog window, then press `Enter`.
```

Comment:
Once published, these instructions will be more difficult than necessary as users will be able to install from their systems. Take a look at other applets for an example.

Reply:
That makes sense, I just copied the one from before I planned on adding it to the store so I forgot to change it haha. Will fix that now